internal/controller/apiextensions/composite/composed.go

```diff
@@ -133,6 +133,27 @@ func (r *APIDryRunRenderer) Render(ctx context.Context, cp resource.Composite, c
 	return errors.Wrap(r.client.Create(ctx, cd, client.DryRunAll), errName)
 }
 
+// An CompositeRenderer renders the composite resource
+type CompositeRenderer struct{} // nolint: golint
```

Comment:
What was golint complaining about here?

Reply:
Package stutter, `composite.CompositeRender`, but the package also contains a few other renderers.